spec/spec_helper.rb

```diff
@@ -26,6 +26,10 @@
   )
 end
 
+VCR.use_cassette('geographical_areas#1013') do
+  GeographicalArea.european_union
+end
+
```

Comment:
I am not sure if we need this.

Reply:
I think I can remove it now that I've removed the mocks on `GeographicalArea.find('1013')`